core/models.py

```diff
@@ -49,6 +50,7 @@ class Meta:
 
     created = models.DateTimeField(default=timezone.now)
     name = models.CharField(max_length=100, unique=True)
+    slug = models.SlugField(unique=True)
```

Comment:
You can have a unique field that's still has multiple null values, right?

I think it's fine (perhaps even clearer) if challenges don't have slugs by default. We can then only add a slug if there's an HTML landing page for it.